docs/guide/content_management.md

```diff
@@ -30,13 +30,13 @@ Under this root there are preset top level Locations in each installation which
 
 ![Content and Media top Locations](img/content_structure_media_library.png)
 
-#### Content
+#### Home
 
-**Content** is the top level Location for the actual contents of a site.
+**Home** is the top level Location for the actual contents of a site.
```

Comment:
Hmm, this is actually more complicated. It's a name of a Content at this top Location. In EE it's "Home", but in OSS it's "eZ Platform". So on an abstract level we can call it "Content". Maybe it should be rephrased so rather abstract term is used, not suggesting any name. Top level location for "Content structure" also would sound good here.